pixi.toml

```diff
@@ -30,6 +29,75 @@ matplotlib = ">=3.9.2"
 [target.linux-aarch64.dependencies]
 blas = { version = "*", build = "netlib" }
 
+[package]
+name = { workspace = true }
+version = { workspace = true }
+
+[package.build]
+backend = { name = "pixi-build-cmake", version = "0.3.*" }
+
+[package.build.config]
+extra-args = [
+  "-DGENERATE_PYTHON_STUBS=ON",
+  "-DBUILD_TESTING=OFF",
+  "-DBUILD_WITH_COLLISION_SUPPORT=ON",
+  "-DBUILD_WITH_ACCELERATE_SUPPORT=OFF",
+  "-DBUILD_WITH_CASADI_SUPPORT=OFF",
+  "-DBUILD_WITH_AUTODIFF_SUPPORT=OFF",
+  "-DBUILD_WITH_EXTRA_SUPPORT=OFF",
+  "-DBUILD_WITH_OPENMP_SUPPORT=OFF",
+  "-DBUILD_WITH_CODEGEN_SUPPORT=OFF",
+  "-DBUILD_WITH_SDF_SUPPORT=OFF",
+  "-DBUILD_PYTHON_BINDINGS_WITH_BOOST_MPFR_SUPPORT=OFF",
+  "-DBUILD_BENCHMARK=OFF",
+  "-DBUILD_TESTING_SCIPY=OFF",
+]
+
+[package.build.target.win-64.config]
+extra-args = [
+  "-DPYTHON_SITELIB=%PREFIX%/Lib/site-packages",
+  "-DGENERATE_PYTHON_STUBS=ON",
+  "-DBUILD_TESTING=OFF",
+  "-DBUILD_WITH_COLLISION_SUPPORT=ON",
+  "-DBUILD_WITH_ACCELERATE_SUPPORT=OFF",
+  "-DBUILD_WITH_CASADI_SUPPORT=OFF",
+  "-DBUILD_WITH_AUTODIFF_SUPPORT=OFF",
+  "-DBUILD_WITH_EXTRA_SUPPORT=OFF",
+  "-DBUILD_WITH_OPENMP_SUPPORT=OFF",
+  "-DBUILD_WITH_CODEGEN_SUPPORT=OFF",
+  "-DBUILD_WITH_SDF_SUPPORT=OFF",
+  "-DBUILD_PYTHON_BINDINGS_WITH_BOOST_MPFR_SUPPORT=OFF",
+  "-DBUILD_BENCHMARK=OFF",
+  "-DBUILD_TESTING_SCIPY=OFF",
+]
+
+[package.build-dependencies]
+pkg-config = ">=0.29.2"
+
+[package.host-dependencies]
+libboost-devel = ">=1.80.0"
+libboost-python-devel = ">=1.80.0"
+eigen = ">=3.4.0"
+numpy = ">=1.22.0"
+python = ">=3.10.0"
+eigenpy = ">=3.7.0"
+urdfdom = ">=4.0"
+hpp-fcl = ">=3"
+
+[package.run-dependencies]
+libboost-devel = ">=1.80.0"
+libboost-python-devel = ">=1.80.0"
```

Comment:
do we have to include these as run dependencies?

Reply:
We should not but it's what we are doing in the conda-forge pinocchio package. Why ? Because consumer project except to be able to do a `find_package(pinocchio)`  without adding more dependencies.
If we don't add `libboost-devel` as a run dependencies, then the `find_package(pinocchio)` will not be able to find boost.

Reply:
Makes sense.